Cauldron/Assets/Scripts/RecipeBook.cs

```diff
@@ -415,6 +428,12 @@ public void SwitchHighlight(RecipeBookEntry recipeBookEntry)
             CloseBook();
             tooltipManager.HighlightRecipe(recipeBookEntry.CurrentRecipe);
             OnSelectRecipe?.Invoke(recipeBookEntry.CurrentRecipe);
+            int eightyPercent = (int)((allMagicalRecipes.Count + allHerbalRecipes.Count) * 0.8);
+            if(PlayerPrefs.GetInt(PrefKeys.AutoCooking) == 0)
```

Comment:
You can use PlayerPrefs.HasKey()
Cool that you found PrefKeys storage and used it, btw)